test/multifile/batch-mode-llvmIRHash-consistency.swift

```diff
@@ -0,0 +1,22 @@
+// Ensure that the LLVMIR hash of the 2nd compilation in batch mode
+// is consistent no matter if the first one generates code or not.
+
+// RUN: %empty-directory(%t)
+// RUN: echo 'public enum E: Error {}' >%t/main.swift
+// RUN: echo >%t/other.swift
```

Comment:
I would prefer of the input files were in test/multifile/batch-mode-inputs/ or something, and not generated with RUN: commands like this.

Reply:
I did have it that way originally. It is our usual custom, but those inputs are typically longer than these. When I realized how short each input was, I decided to do it this way because incorporating the inputs here makes it easier for someone to understand what the test does. The other way, a person would have to:
1. navigate to `Inputs`, 
2. navigate to the sub-folder, remembering the name of the test in question, and 
3. open the files and read them.

But, I'm likely missing something: What is the advantage of having separate input files in the `Inputs` subdirectory?